logback-site/src/site/pages/css/screen.css

```diff
@@ -217,4 +217,54 @@ td:hover .anchor {
   text-decoration: none;
 }
 
+/**
+ * Code added for mobile compatibility
+ */
+.slide_close { display: none; }
+.handle { display: none; }
 
+@media only screen
+and (max-width: 1175px) {
+  #right,#left {
+    display: none;
+    position: fixed;
+  }
+  #right:target,#left:target {
+    display: block;
+  }
+  #right:target+#right_open::after,#left:target+#left_open::after {
+    content: '';
+  }
+  .handle {
+    display: inline-block;
+    width: 50%;
+    height: 1.5em;
+    background-color: #FFE0B0;
+    border: thin solid #cccccc;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Using height with border can sometimes make elements larger than you expect.](https://www.codacy.com/app/tony19/logback/file/2375991531/issues/source?bid=3222618&fileBranchId=3222700#l243)